### DISCUSSION TITLE
function_exists ( 'ray' )

### DISCUSSION THREAD

**Discussion by carlituxman** [spatie/wordpress-ray#37] [Ideas]
It would be necessary something like this using mu-plugins
`if ( !function_exists('ray') )
{
    function ray() {}
}
`
for production environments

**Comment by freekmurze** [comment]
I don't understand your remark, but feel free to PR non-breaking improvements.

**Reply by carlituxman** [reply]
I use mu-plugins, then at production hasn't plugin. Before upload changes to environment production need quit ray() on my code, with this don't needed
